Tools/plot_parallel.py

```diff
@@ -20,37 +24,39 @@
 This script also proposes an option to plot one quantity over all timesteps.
 The data of all plotfiles are gathered to rank 0, and the quantity evolution
 is plotted and saved to file. For the illustration, this quantity is the max
-of Ey. Use " --plot_Ey_max_evolution " to activate this option.
+of Ey. Use " --plot_max_evolution Ey" to activate this option.
 
 To get help, run
 > python plot_parallel --help
 '''
 
 # Parse command line for options.
 parser = argparse.ArgumentParser()
-parser.add_argument('--path', dest='path', default='.',
+parser.add_argument('--path', default='diags/plotfiles',
                     help='path to plotfiles. Plotfiles names must be plt?????')
-parser.add_argument('--plotlib', dest='plotlib', default='yt',
+parser.add_argument('--plotlib', default='yt',
                     choices=['yt','matplotlib'],
                     help='Plotting library to use')
-parser.add_argument('--field', dest='field', default='Ez',
+parser.add_argument('--field', default='Ez',
                     help='Which field to plot, e.g., Ez, By, jx or rho. The central slice in y is plotted')
-parser.add_argument('--pjump', dest='pjump', default=20,
+parser.add_argument('--pjump', default=20,
                     help='When plotlib=matplotlib, we plot every pjump particle')
-parser.add_argument('--use_vmax', dest='use_vmax', default=False,
-                    help='Whether to put bounds to field colormap')
-parser.add_argument('--vmax', dest='vmax', default=1.e12,
-                    help='If use_vmax=True, the colormab will have bounds [-vamx, vmax]')
-parser.add_argument('--slicewidth', dest='slicewidth', default=10.e-6,
+parser.add_argument('--vmax', type=float, default=None,
+                    help='If specified, the colormap will have bounds [-vmax, vmax]')
+parser.add_argument('--slicewidth', default=10.e-6,
                     help='Only particles with -slicewidth/2<y<slicewidth/2 are plotted')
-parser.add_argument('--parallel', dest='parallel', action='store_true', default=False,
+parser.add_argument('--parallel', action='store_true', default=False,
                     help='whether or not to do the analysis in parallel (e.g., 1 plotfile per MPI rank)')
 parser.add_argument('--species', dest='pslist', nargs='+', type=str, default=None,
                     help='Species to be plotted, e.g., " --species beam plasma_e ". By default, all species in the simulation are shown')
-parser.add_argument('--plot_Ey_max_evolution', dest='plot_Ey_max_evolution', action='store_true', default=False,
-                    help='Whether to plot evolution of max(Ey) to illustrate how to plot one quantity across all plotfiles.')
+parser.add_argument('--plot_max_evolution', type=str, default=None,
+                    help='Quantity to plot the max of across all data files')
```

Comment:
Right now I am using the script to plot the evolution of particle quantities... Do you think we should rename this option and the functions involved with generic names, e.g., `--plot_evolution` and replace function `get_field_max` by `evolution_get_quantity` etc.?
If so, that could be included in this PR or we can make another PR for that (or just create an issue).

Reply:
Yes, the names can be changed to be generic. I'll make the change.